ledger/sandbox-on-x/src/main/scala/com/daml/ledger/sandbox/bridge/LedgerBridge.scala

```diff
@@ -128,7 +128,7 @@ object LedgerBridge {
     val party = s.hint.getOrElse(UUID.randomUUID().toString)
     Update.PartyAddedToParticipant(
       party = Ref.Party.assertFromString(party),
-      displayName = s.displayName.getOrElse(party),
+      displayName = s.displayName.getOrElse(""),
```

Comment:
o..O well spotted!

Reply:
Seems like something that we might want to have a test for somewhere?

Reply:
Test added